alarm/utils.py

```diff
@@ -33,3 +30,128 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def refresh_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in Threshold.objects.filter(trade_pair=trade_pair):
+        if ThresholdBrake.objects.filter(threshold=threshold).exists():
+            continue
+    for threshold in thresholds:
+        try:
+            latest_threshold_brake = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair).latest(
+                'happened_at')
+            if latest_threshold_brake:
+                return False
+        except ThresholdBrake.DoesNotExist:
+            pass
+
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def create_message_about_threshold_break():
+    all_threshold_breaks = ThresholdBrake.objects.all()
```

Comment:
That will inform about all threshold, while we need only thresholds, which users are not aware of. You can say, that we erase old thresholds. But how will we debug a program later then?

You can say, that we make a clean up for candles, but they will flood a DB very fast, as they are created every second. Keep broken thresholds, and here use only new for a user brakes.

Reply:
++